### PR TITLE
Fix back button so that it triggers a full page reload when forceRefresh is enabled

### PR DESCRIPTION
**Pull request by reywright**
https://github.com/ReactTraining/history/issues/638

### CODE REVIEW

**Comment by reywright** [comment]
Wasn't "really" sure how best to describe the fix in `CHANGES.md` but fixing that was my major goal so I described it that way. Let me know if I can state it a better way.

File: CHANGES.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+## HEAD
+> Nov 03, 2018
+
+- Browser back button now triggers a full page reload when forceRefresh is enabled
```
Comment (pshrmn) [inline CHANGES.md]:
```suggestion
- History API is not used when `forceRefresh` is enabled
```

Comment (reywright) [reply CHANGES.md]: Thanks

**Comment by murphyjt** [comment]
Is it possible to merge this change? I have the same issue @reywright has in https://github.com/ReactTraining/history/issues/638

**Comment by reywright** [comment]
This PR is 2 months away from its one year anniversary. I'd solve the diff the here but no one has communicated anything about wanting to get this in, and I'm not going to resolve the conflicts just so it can sit here another year unfortunately.

Screen Shot 2019-08-28 at 9 26 01 PM

The above is a screenshot of the diff, if anyone's actually interested in getting this in.

**Pull request closed without merging**